src/lms-harmonizer/edfi_lms_harmonizer/exceptions_reports.py

```diff
@@ -91,6 +54,32 @@ def print_summary(adapter: Adapter) -> None:
     else:
         logger.info("There are no unmatched sections or users in the database.")
```

Comment:
We probably don't need to tell people this, as it is an assumption they will make. But good to include it in the debug output. 

```suggestion
        logger.debug("There are no unmatched sections or users in the database.")
```